src/averageAges.js

```diff
@@ -54,9 +66,29 @@ function calculateWomenAverageAge(people, withChildren) {
  *
  * @return {number}
  */
+
+function calculateAvg(arr) {
+  const total = arr.reduce((a, b) => a + b, 0);
```

Comment:
And also here, total => totalSum, total => sumOfNumbers
and something more descriptive instead of a and b